From b12f77638b757462c9f0a47dd1d911a6d28faa51 Mon Sep 17 00:00:00 2001
From: Nizan Zorea <nzorea@marvell.com>
Date: Tue, 14 Jun 2016 14:20:34 +0300
Subject: [PATCH 1192/1240] fix: config: remove ihb from 70x0 palladium

initialization of the ihb phy causes hang during boot time.
Note: commit c7a1234587f5bd72a6145bab5ec9c97c53cfec80 fixes the problem
but since ihb phy access is not needed in Palladium,
we removed this initialization

Change-Id: I41b39a000ee4a4ff4678ab870fb3345188b53443
Signed-off-by: Nizan Zorea <nzorea@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30457
Reviewed-by: Haim Boot <hayim@marvell.com>
Tested-by: Haim Boot <hayim@marvell.com>
---
 configs/mvebu_armada70x0_pd_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/mvebu_armada70x0_pd_defconfig b/configs/mvebu_armada70x0_pd_defconfig
index 2a1de8c..9bfa4c6 100644
--- a/configs/mvebu_armada70x0_pd_defconfig
+++ b/configs/mvebu_armada70x0_pd_defconfig
@@ -18,6 +18,7 @@ CONFIG_CMD_MVEBU_BUBT=y
 +S:CONFIG_OF_CONTROL=y
 +S:CONFIG_OF_EMBED=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="armada-70x0-pd"
+# CONFIG_MVEBU_PHY_INDIRECT is not set
 CONFIG_MVEBU_CCU=y
 CONFIG_MVEBU_IOB=y
 CONFIG_MVEBU_RFU=y
-- 
1.9.1

